src/test/web/messages/message-store.properties

```diff
@@ -13,6 +13,6 @@
 # either express or implied. See the License for the specific
 # language governing permissions and limitations under the License.
 #
-[ERROR]=Error /message-store
-[INFO]=Info /message-store
-[WARNING]=Warning /message-store
\ No newline at end of file
+[STORE_ERROR]=Error /message-store
+[STORE_INFO]=Info /message-store
+[STORE_WARNING]=Warning /message-store
```

Comment:
Was this change just to make it simpler to understand?

Reply:
It seems like this change made `GlobalTest.get_message_callback` better because it looked like what it's trying to test there is that the message store has the right values AND is scoped correctly (meaning not a singleton, etc.). I say that because of the fact that `CallbackAction` is making an HTTP request to another action (`/message-store`). Since the objective of the test seemed to be to ensure that CallbackAction's messages were in the store and not `MessageStoreAction`, changing the keys made sense since it's easier to assert the keys in the message store than the message contents.